modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/index.js

```diff
@@ -28,11 +33,59 @@ const InvitesModal = ({observer, onClose, project}) => {
 };
 
 const TeamMembers = () => {
-	const {project} = useOutletContext();
+	const [dxpCloudEnvironment, setDxpCloudEnvironment] = useState();
+	const activationStatusAC = 'Active';
+	const groupId = 'groupid';
+	const {project, subscriptionGroups} = useOutletContext();
 	const [visible, setVisible] = useState(false);
 	const modalProps = useModal({
 		onClose: () => setVisible(false),
 	});
+	const [activatedStatusDXPC, setActivatedStatusDXPC] = useState();
+	const [activatedStatusAC, setActivatedStatusAC] = useState();
+	const [activatedLinkDXPC, setActivatedLinkDXPC] = useState();
+	const [activatedLinkAC, setActivatedLinkAC] = useState();
+
+	useEffect(() => {
+		const getOnboardingFormData = async () => {
+			const {data} = await client.query({
+				query: getDXPCloudEnvironment,
+				variables: {
+					filter: `accountKey eq '${project.accountKey}'`,
+					scopeKey: Liferay.ThemeDisplay.getScopeGroupId(),
+				},
+			});
+
+			if (data) {
+				const dxpProjectId =
+					data.c?.dXPCloudEnvironments?.items[0]?.projectId;
+
+				setDxpCloudEnvironment(dxpProjectId);
+			}
+		};
+		getOnboardingFormData();
+
+		const accountSubscriptionFromDXPC = subscriptionGroups.find(
+			(subscriptionGroup) =>
+				subscriptionGroup.name === PRODUCT_TYPES.dxpCloud
+		);
+		const activationStatusDXPC =
+			accountSubscriptionFromDXPC?.activationStatus;
+
+		if (activationStatusDXPC === STATUS_TAG_TYPE_NAMES.active) {
+			setActivatedStatusDXPC(activationStatusDXPC);
+		}
+		if (activationStatusAC === STATUS_TAG_TYPE_NAMES.active) {
+			setActivatedStatusAC(activationStatusAC);
+		}
+
+		setActivatedLinkDXPC(
+			`https://console.liferay.cloud/projects/${dxpCloudEnvironment}/overview`
+		);
+		setActivatedLinkAC(
+			`https://analytics.liferay.com/workspace/${groupId}/sites`
+		);
```

Comment:
Hi Lucas, I think this link should be a constant. You are using the constant "groupId" to build the link and it will be the same ever.

Reply:
So, this state isn`t necessary, just declare a constant.

Reply:
Like this:

`const activatedLinkAC = https://analytics.liferay.com/workspace/groupid/sites`

Reply:
That are fixed, but it will change when Analytics Cloud story will be done.